apps/bfd-model/bfd-model-rif/src/main/resources/db/scripts/Query_synthetic_id_ranges.sql

```diff
@@ -0,0 +1,102 @@
+-- Find the beneficiaries.bene_id range to avoid collisions with synthetic data.
+WITH ids AS (
```

Comment:
above this query in addition to you original comment please mention that when generating new synthetic data, a gap in the ranges returned by this query should be found that is big enough and is preferable contiguous with the last range used.